### ISSUE TITLE
[BUG]: Warning given during face_encodings call: "pkg_resources is deprecated as an API."

### ISSUE THREAD

**Issue by robblovell** [serengil/deepface#1490]
### Before You Report a Bug, Please Confirm You Have Done The Following...

- [x] I have updated to the latest version of the packages.
- [x] I have searched for both [existing issues](https://github.com/serengil/deepface/issues) and [closed issues](https://github.com/serengil/deepface/issues?q=is%3Aissue+is%3Aclosed) and found none that matched my issue.

### DeepFace's version

0.0.93

### Python version

3.12.10

### Operating System

OS/X

### Dependencies

pip freeze
absl-py==2.3.1
astunparse==1.6.3
attrs==25.3.0
awscli==1.41.3
beautifulsoup4==4.13.4
blinker==1.9.0
boto3==1.39.3
botocore==1.39.3
cachetools==6.1.0
certifi==2025.6.15
chardet==5.2.0
charset-normalizer==2.0.12
click==8.2.1
cmake==4.0.3
colorama==0.4.6
DateTime==5.5
deepface==0.0.93
distlib==0.3.9
dlib==19.24.9
dnspython==2.7.0
docker==7.1.0
docutils==0.19
face-recognition==1.3.0
face_recognition_models==0.3.0
filelock==3.18.0
fire==0.7.0
Flask==3.1.1
flask-cors==6.0.1
flatbuffers==25.2.10
gast==0.6.0
gdown==5.2.0
google-pasta==0.2.0
grpcio==1.73.1
gunicorn==23.0.0
h5py==3.14.0
idna==3.10
imutils==0.5.4
iniconfig==2.1.0
IPTCInfo3==2.1.4
itsdangerous==2.2.0
Jinja2==3.1.6
jmespath==1.0.1
joblib==1.5.1
jsonpickle==4.1.1
kairos_face_recognition_lib @ git+https://github.com/ffmmjj/kairos-face-sdk-python@f94edc0ebcd2ca6a264c1f3296e6ccc5190de601
keras==3.10.0
libclang==18.1.1
lz4==4.4.4
Markdown==3.8.2
markdown-it-py==3.0.0
MarkupSafe==3.0.2
mdurl==0.1.2
merry==0.3.0
ml_dtypes==0.5.1
mtcnn==1.0.0
mypy==1.16.1
mypy_extensions==1.1.0
namex==0.1.0
numpy==2.1.3
opencv-python==4.12.0.88
opt_einsum==3.4.0
optree==0.16.0
packaging==25.0
pandas==2.3.1
pathspec==0.12.1
pillow==11.3.0
platformdirs==4.3.8
pluggy==1.6.0
protobuf==5.29.5
psycopg2-binary==2.9.10
py==1.11.0
pyasn1==0.6.1
pydevd-pycharm==252.23309.36
pyflakes==3.4.0
Pygments==2.19.2
pymongo==4.13.2
pyproject-api==1.9.1
PySocks==1.7.1
pytest==8.4.1
python-dateutil==2.9.0.post0
python-dotenv==1.1.1
pytz==2025.2
PyYAML==5.3.1
requests==2.32.4
retina-face==0.0.17
rich==14.0.0
rsa==4.7.2
s3transfer==0.13.0
scikit-learn==1.7.0
scipy==1.16.0
setuptools==80.9.0
six==1.17.0
soupsieve==2.7
tensorboard==2.19.0
tensorboard-data-server==0.7.2
tensorflow==2.19.0
termcolor==3.1.0
tf_keras==2.19.0
threadpoolctl==3.6.0
tomli==2.2.1
tox==4.27.0
tqdm==4.67.1
types-requests==2.28.11
types-urllib3==1.26.25.14
typing_extensions==4.14.1
tzdata==2025.2
urllib3==1.26.20
virtualenv==20.31.2
watchdog==6.0.0
websocket-client==1.8.0
Werkzeug==3.1.3
wheel==0.45.1
wrapt==1.17.2
zope.interface==7.2

### Reproducible example

```Python
from face_recognition import face_encodings


def get_face_embeddings(images):
    encodings = []
    failed_encodings = []
    for i in range(len(images)):
        encoding = face_encodings(images[i])
        if len(encoding) > 0:
            encodings.append(encoding[0])
        else:
            failed_encodings.append(i)
    return encodings, failed_encodings
```

### Relevant Log Output

UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
  from pkg_resources import resource_filename

### Expected Result

No warning is printed.

### What happened instead?

Code works, but a warning is printed. pkg_resources will be removed in November, is there a plan to replace it with importlib.metadata?

**Comment by serengil** [comment]
You are not using even deepface!
